etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/TransformationRuleEngineIntegrationTest.groovy

```diff
@@ -0,0 +1,153 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation
+
+import gov.hhs.cdc.trustedintermediary.FhirBundleHelper
+import gov.hhs.cdc.trustedintermediary.ExamplesHelper
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.RuleLoader
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirImplementation
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiFhirResource
+import gov.hhs.cdc.trustedintermediary.external.jackson.Jackson
+import gov.hhs.cdc.trustedintermediary.wrappers.HapiFhir
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetadata
+import gov.hhs.cdc.trustedintermediary.wrappers.formatter.Formatter
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.Patient
+import spock.lang.Specification
+
+class TransformationRuleEngineIntegrationTest extends Specification {
+    def engine = TransformationRuleEngine.getInstance("transformation_definitions.json")
+    def fhir = HapiFhirImplementation.getInstance()
+    def mockLogger = Mock(Logger)
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+
+        TestApplicationContext.register(Formatter, Jackson.getInstance())
+        TestApplicationContext.register(HapiFhir, fhir)
+        TestApplicationContext.register(TransformationRuleEngine, engine)
+        TestApplicationContext.register(RuleLoader, RuleLoader.getInstance())
+        TestApplicationContext.register(Logger, mockLogger)
+        TestApplicationContext.register(MetricMetadata, Mock(MetricMetadata))
+
+        TestApplicationContext.injectRegisteredImplementations()
+    }
+
+    def "transformation rules run without error"() {
```

Comment:
I believe this is the test that is leaking onto the test below (`all transformations in the definitions file have existing custom methods`).